My Simple App/Business Logic/Location.swift

```diff
@@ -10,14 +10,14 @@ import Foundation
 import CoreLocation
 
 class Location {
-  static func coordinates(from address: String, completion: @escaping (CLLocation?) -> Void) {
+  static func placemarker(from address: String, completion: @escaping (CLPlacemark?) -> Void) {
```

Comment:
Wdyt about replacing the CoreLocation (`CLPlacemark`) type into its own struct?

That way everyone using Location doesn't need to import CoreLocation